config/locales/en.yml

```diff
@@ -44,6 +44,7 @@ en:
         docs: Documentation URL
         mail: Mailing List URL
         wiki: Wiki URL
+        funding: Funding URL
```

Comment:
Please add this key to all locale files. value for everything beside `en` will be empty.